package-lock.json

```diff
@@ -32829,7 +32829,7 @@
       "dependencies": {
         "@microsoft/fast-colors": "^5.3.1",
         "@microsoft/fast-element": "^1.10.1",
```

Comment:
Looks like the lock file is already pulling in `1.12.0` which is fine, but may as well bump this to align (and make sure to re-run npm install for any package-lock updates)

```
        "@microsoft/fast-element": "^1.12.0",
```